_includes/content.html

```diff
@@ -40,20 +40,31 @@
 <h2>Defaults</h2>
 
 <p>By default Griddler will look for a class to be created in your application
-called <code>EmailProcessor</code> with a class method named
-<code>process</code>, taking in one argument, the <code>Griddler::email</code>
-to process.  For example, in <code>./lib/email_processor.rb</code>:</p>
+called <code>EmailProcessor</code> with an instance method named
```

Comment:
Right now on Griddler.io the docs read:

![screen shot 2015-03-05 at 10 11 48 am](https://cloud.githubusercontent.com/assets/463175/6507460/1fdf4a7e-c320-11e4-913d-04288218ab79.png)

This bit me, since not only does Griddler now not use a Class level method (process vs self.process), but there's really no mention of initializing in an email through another method. Compare to the readme:

![screen shot 2015-03-05 at 10 11 57 am](https://cloud.githubusercontent.com/assets/463175/6507490/4b00afea-c320-11e4-84a4-a5712325ff73.png)